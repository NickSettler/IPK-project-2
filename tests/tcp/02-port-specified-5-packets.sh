./ipk-sniffer -i en0 --tcp -p 443 -n 5