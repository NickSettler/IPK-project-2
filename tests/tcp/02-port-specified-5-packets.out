timestamp: 2023-04-17T19:51:58.164+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 98:77:e7:50:60:5e
frame length: 97 bytes
src IP: 192.168.1.17
dst IP: 54.154.225.175
src port: 58798
dst port: 443

0x0000: 98 77 e7 50 60 5e b0 be 83 4a c4 b0 08 00 45 00  .w.P`^...J....E.
0x0010: 00 53 00 00 40 00 40 06 60 a2 c0 a8 01 11 36 9a  .S..@.@.`.....6.
0x0020: e1 af e5 ae 01 bb cb 1e a0 8d 3c 03 c6 b7 80 18  ..........<.....
0x0030: 08 07 0c a5 00 00 01 01 08 0a 2e db 5d b3 bb d4  ............]...
0x0040: aa 91 15 03 03 00 1a 00 00 00 00 00 00 00 03 67  ...............g
0x0050: 98 dd 04 de ce 29 6c 26 79 64 fd c0 dd f0 bc 94  .....)l&yd......
0x0060: 20                                                

timestamp: 2023-04-17T19:51:58.164+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 98:77:e7:50:60:5e
frame length: 66 bytes
src IP: 192.168.1.17
dst IP: 54.154.225.175
src port: 58798
dst port: 443

0x0000: 98 77 e7 50 60 5e b0 be 83 4a c4 b0 08 00 45 00  .w.P`^...J....E.
0x0010: 00 34 00 00 40 00 40 06 60 c1 c0 a8 01 11 36 9a  .4..@.@.`.....6.
0x0020: e1 af e5 ae 01 bb cb 1e a0 ac 3c 03 c6 b7 80 11  ..........<.....
0x0030: 08 07 4b cd 00 00 01 01 08 0a 2e db 5d b3 bb d4  ..K.........]...
0x0040: aa 91                                            ..

timestamp: 2023-04-17T19:51:58.200+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 98:77:e7:50:60:5e
frame length: 78 bytes
src IP: 192.168.1.17
dst IP: 18.203.57.69
src port: 58803
dst port: 443

0x0000: 98 77 e7 50 60 5e b0 be 83 4a c4 b0 08 00 45 00  .w.P`^...J....E.
0x0010: 00 40 00 00 40 00 40 06 2c ef c0 a8 01 11 12 cb  .@..@.@.,.......
0x0020: 39 45 e5 b3 01 bb c8 69 7b 87 00 00 00 00 b0 02  9E.....i{.......
0x0030: ff ff c2 d9 00 00 02 04 05 b4 01 03 03 06 01 01  ................
0x0040: 08 0a ed b0 4d 48 00 00 00 00 04 02 00 00        ....MH........

timestamp: 2023-04-17T19:51:58.228+02:00
src MAC: 98:77:e7:50:60:5e
dst MAC: b0:be:83:4a:c4:b0
frame length: 66 bytes
src IP: 54.154.225.175
dst IP: 192.168.1.17
src port: 443
dst port: 58798

0x0000: b0 be 83 4a c4 b0 98 77 e7 50 60 5e 08 00 45 00  ...J...w.P`^..E.
0x0010: 00 34 25 b2 40 00 d8 06 a3 0e 36 9a e1 af c0 a8  .4%.@.....6.....
0x0020: 01 11 01 bb e5 ae 3c 03 c6 b7 cb 1e a0 ad 80 11  ......<.........
0x0030: 00 78 34 51 00 00 01 01 08 0a bb d4 c9 9b 2e db  .x4Q............
0x0040: 5d b3                                            ].

timestamp: 2023-04-17T19:51:58.228+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 98:77:e7:50:60:5e
frame length: 66 bytes
src IP: 192.168.1.17
dst IP: 54.154.225.175
src port: 58798
dst port: 443

0x0000: 98 77 e7 50 60 5e b0 be 83 4a c4 b0 08 00 45 00  .w.P`^...J....E.
0x0010: 00 34 00 00 40 00 40 06 60 c1 c0 a8 01 11 36 9a  .4..@.@.`.....6.
0x0020: e1 af e5 ae 01 bb cb 1e a0 ad 3c 03 c6 b8 80 10  ..........<.....
0x0030: 08 07 2c 81 00 00 01 01 08 0a 2e db 5d f4 bb d4  ..,.........]...
0x0040: c9 9b                                            ..

